Dockerfile

```
@@ -28,8 +28,11 @@ RUN apt -q -qq update && \
 RUN rosdep update
 
 RUN mkdir -p /catkin_ws/src && cd /catkin_ws/src && \
-  git clone --depth 1 https://github.com/IntelRealSense/realsense-ros.git && \
-  git clone --depth 1 https://github.com/pal-robotics/ddynamic_reconfigure
+  git clone https://github.com/IntelRealSense/realsense-ros.git && \
+  cd realsense-ros && \
+  git checkout ros1-legacy && \
+  cd .. && \
```

Your changes work for me, thank you. You could streamline the command a little like this: 

```suggestion
  git clone --branch ros1-legacy https://github.com/IntelRealSense/realsense-ros.git && \
```